PLANNER-2050 Reformat the upgrade recipe for 8 (#250)
kiegroup/optaplanner-website: download/upgradeRecipe/upgradeRecipe8.adoc

//
//We currently intend to support a minimal version of Java 11 throughout the entire 8.x series.

[.upgrade-recipe-major.upgrade-recipe-public-api]
=== `SolverFactory` and `PlannerBenchmarkFactory` no longer support KIE containers

> With OptaPlanner's move towards Kogito, the KIE container concept no longer applies.

Reviewer: ```suggestion
Due to OptaPlanner aligning to Kogito, the KIE container concept no longer applies.
```
